podman/tests/integration/test_manifests.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from contextlib import suppress
+
+from podman import PodmanClient
+from podman.errors.exceptions import APIError, ImageNotFound
+from podman.tests.integration import base
+
+# @unittest.skipIf(os.geteuid() != 0, 'Skipping, not running as root')
+
+
+class ManifestsIntegrationTest(base.IntegrationTest):
+    def setUp(self):
+        super().setUp()
+
+        self.client = PodmanClient(base_url=self.socket_uri)
+        self.addCleanup(self.client.close)
+
+        self.alpine_image = self.client.images.pull("quay.io/libpod/alpine", tag="latest")
+
+    def tearDown(self) -> None:
+        self.client.images.remove(self.alpine_image, force=True)
+        with suppress(ImageNotFound):
+            self.client.images.remove("quay.io/unittest/alpine:latest", force=True)
+
+    def test_manifest_crud(self):
+        """Test Manifest CRUD."""
+
+        self.assertFalse(
+            self.client.manifests.exists("quay.io/unittest/alpine:latest"),
+            "Image store is corrupt from previous run",
+        )
+
+        with self.subTest("Create"):
+            manifest = self.client.manifests.create(["quay.io/unittest/alpine:latest"])
```

Comment:
Would it make sense to deliberately force an invalid name here?  For example: `does.not/exist/alpine:latest`

Reply:
I would also encourage use of a object variable, instead of duplicating the same test-name over and over.

Reply:
In tests, I prefer to duplicate the text when it provides context to the test. I've been burnt too many times otherwise.